src/System.Diagnostics.DiagnosticSource/ref/Configurations.props

```diff
@@ -2,9 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      net45;
       netstandard;
-      netstandard1.1;
```

Comment:
Did you figure out why the netstandard1.1 configuration was picked? I'm mainly interested because we might need to add it back at some point.